### PR TITLE
🔧[e2e cbt] setup retry

### PR DESCRIPTION
**Pull request by bcaudan**

### CODE REVIEW

File: scripts/cbt-with-retry.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 'use strict'
 
-const path = require('path')
 const exec = require('child_process').exec
 
-const rootDirectory = path.join(__dirname, '..')
-const COMMAND = `${rootDirectory}/node_modules/.bin/karma start ${rootDirectory}/test/unit/karma.cbt.conf.js`
+const [_, __, ...args] = process.argv
```
Comment (BenoitZugmeyer) [inline scripts/cbt-with-retry.js]:
```suggestion
const args = process.argv.slice(2)
```

File: scripts/cbt-with-retry.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+const COMMAND = `yarn ${args.join(' ')}`
+const RETRY_DELAY_IN_MIN = 5
```
Comment (BenoitZugmeyer) [inline scripts/cbt-with-retry.js]: 5 minutes seems a bit much?

File: scripts/cbt-with-retry.js [hunk 3]
```diff
@@ -27,7 +30,11 @@ function executeWithRetry() {
 }
 
 function isRetryAllowed(logs) {
-  return logs.indexOf('UnhandledRejection') !== -1
+  return (
+    logs.indexOf('UnhandledRejection') !== -1 ||
```
Comment (BenoitZugmeyer) [inline scripts/cbt-with-retry.js]:
```suggestion
    logs.includes('UnhandledRejection') ||
```

File: scripts/cbt-with-retry.js (hunk 2 above)
Comment (bcaudan) [reply scripts/cbt-with-retry.js]: unit takers 6-7 min, what do you think?

Comment (BenoitZugmeyer) [reply scripts/cbt-with-retry.js]: Ok it makes sense.

**Pull request merged**